app/static/src/app/components/header/AppHeader.vue

```diff
@@ -66,14 +74,22 @@ export default defineComponent({
             return sessionLabel.value ? `Session: ${sessionLabel.value}` : "Sessions";
         });
 
+        const languagesKeys: LanguagesKeys = {
+            [Language.en]: "English",
```

Comment:
We can just add to this object and it will show more items in the drop down, however we do have to make sure that the translations for those keys are registered, there will be some degree of baking these things into wodin, I think setting what languages should be available is a great idea for a future config PR actually! But I do wonder if we will ever have that many languages?